packages/loader/container-loader/dev.md

```diff
@@ -0,0 +1,30 @@
+# @fluidframework/container-loader
+
+This document was created to provide condensed readable summaries of the more nuanced inner workings of the container-loader service. 
+
+**Topics covered below:**
+- [@fluidframework/container-loader](#fluidframeworkcontainer-loader)
+  - [ConnectionState Change Transitions](#connectionstate-change-transitions)
+    - [What happens when a client gets disconnected?](#what-happens-when-a-client-gets-disconnected)
+    - [Join & Leave Ops](#join--leave-ops)
+
+## ConnectionState Change Transitions
+
+A Delta Manager class is used to handle connect, disconnect, throttle and other events.
+
+### What happens when a client gets disconnected?
+
+If a client (Client A) gets disconnected for any reason, the prevClientLeftTimer is started. This means we wait for some time for the server to broadcast a leave op for Client A. This only happens once to prevent any timer resets on multiple disconnect events. 
+
+NOTE: A client that disconnects can NOT reconnect again with the same clientID. Client IDs are assigned when socket round trip is successful.
```

Comment:
I feel like this document has some overlap with two types of documents:
- readme.md that is next to it. Take a look - it talks about connections and has sections for different types of developers. I'd prefer to augment it with any data that stays at the level of API / expectations (not protocol, i.e. discussions about specific ops likely does not belong there).
- documentation on FluidFramework.com

We lack documentation about protocol and protocol requirements / expectations. I'd rather find a place to start putting this documentation (all content about ops, signals, etc.). Feels like FluidFramework.com is slightly better place for this IMHO.

@ssimic2, @GaryWilber, @tylerbutler  for their thoughts.

Reply:
I agree that ff.com is a good place for this stuff to begin to show up. However, the site isn't currently structured in a way to cover the high level FRS-focused entry-points and the lower level APIs that more deeply Fluid-integrated apps need to use. @Josmithr has done some thinking about how to expose our full API docs on the site, and this seems somewhat related.